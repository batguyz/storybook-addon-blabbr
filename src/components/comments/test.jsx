import React from 'react';
import renderer from 'react-test-renderer'; // eslint-disable-line
import Comments from './';

test('Comments render correctly', () => {
  const tree = renderer.create(
    <Comments
      comments={[]}
      commentIdBeingEdited="commentId123"
      userCommentBeingUpdated="updating comment"
      currentUser="currentUser"
      onUserCommentUpdate={() => {}}
      onUserCommentEdit={() => {}}
      onUserCommentEditSave={() => {}}
      onUserCommentEditCancel={() => {}}
      onUserCommentDelete={() => {}}
      isShowingAllComments
      onShowAllComments={() => {}}
    />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
