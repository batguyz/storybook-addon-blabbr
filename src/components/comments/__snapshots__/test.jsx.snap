exports[`test Comments render correctly 1`] = `
<div>
  <div>
    <p>
      No comments to show for this story
    </p>
  </div>
</div>
`;
